src/django/api/mail.py

```diff
@@ -23,6 +24,17 @@ def make_facility_url(request, facility):
     )
 
 
+def make_facility_list_url(list_id):
+    if settings.ENVIRONMENT == 'Development':
+        protocol = 'http'
+        host = 'localhost:6543'
+    else:
```

Comment:
When I tested this on staging I realized that the link in the email pointed to production rather than staging. For correctness and ease of testing we should consider something like this:

```python
if settings.ENVIRONMENT == 'Development':
    protocol = 'http'
    host = 'localhost:6543'
else:
    protocol = 'https'
    if settings.ENVIRONMENT == 'Staging':
        host = 'staging.openapparel.org
    else:
        host = 'openapparel.org'
```

Reply:
Good catch, thanks!